modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -71,7 +71,15 @@ def full_body(action:, body:, namespace:)
 
     def parsed_response(res, action, key = nil)
       parsed = Hash.from_xml(res.body)
-
+      if action == 'findIntentToFileByPtcpntIdItfTypeCd'
+        itf_response = []
+        parsed.dig('Envelope', 'Body', "#{action}Response", key).each do |itf|
```

Comment:
BGS likes to return a hash if one and an array if multiple. I'm not sure this is guaranteed, but I'm wondering if we'll see an error here. Looks like a `.each` will work, but the next line will break. May want to check in on this if possible. Dunno how we'll find one that could return a hash though. 🤔 

```
{a: 'b', b: 'c'}.each { |a| a.deep_transform_keys(&:underscore) }
(irb):3:in `block in <main>': undefined method `deep_transform_keys' for [:a, "b"]:Array (NoMethodError)
```